### PR TITLE
FileReader: wrapper for window.FileReader that provides Promise support

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

**Comment by mrjoelkemp** [comment]
:+1:

File: test/specs/dom/FileReader.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          expect(imageData.name).toBe(image.name);
+          expect(imageData.mode).toBe(image.mode);
+          expect(imageData.source).toBe(image.source);
+          expect(imageData.result).toBe(image.result);
+          done();
+        });
+      });
+    });
+
+    describe('promse', function() {
```
Comment (jrefano) [inline test/specs/dom/FileReader.js]: typo `promise`

**Comment by jrefano** [comment]
:+1:

**Comment by Attamusc** [comment]
:+1:

**Pull request merged**